### ISSUE TITLE
can not import [v0.16.0]

### ISSUE THREAD

**Issue by Yujie95** [nschloe/quadpy#378]
I tried to import quadpy after installing the latest version(v0.16.0) and had this error:
cannot import name 'concat' from 'quadpy.t2._helpers' (C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\site-packages\quadpy\t2\_helpers.py)

I then switched to v0.15.2, everything is fine again.

**Comment by nschloe** [comment]
Cannot reproduce.

**Comment by gabhinav0110** [comment]
I am having the same problem even after I switched to v0.15.2

**Comment by gabhinav0110** [comment]
Kept on trying older versions and finally, this problem was not present in v0.14.11

**Comment by nschloe** [comment]
Looks like a broken installation. This line
```
----> 5 from ._helpers import T2Scheme, concat, expand_symmetries, s1, s2, s3
```
is not present in `quadpy/t2/_lyness_jespersen.py`, see https://github.com/nschloe/quadpy/blob/master/quadpy/t2/_lyness_jespersen.py.

**Comment by gabhinav0110** [comment]
I see that, but I literally did `pip install quadpy`

**Comment by nschloe** [comment]
The ways in which Python installations can break amaze me every day. You should check _where_ the data installs. Perhaps a new location, and the old one is preferred? Perhaps python is picking up quadpy from the local folder? You can only debug it yourself.

**Comment by nschloe** [comment]
I see this issue in one of my other projects now. Investigating...
https://github.com/nschloe/optimesh/pull/59/checks?check_run_id=1028513670

**Comment by nschloe** [comment]
Fixed in 0.16.1.
